packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -303,42 +430,71 @@ describe('meeting', () => {
       expect(() => meeting.requestStopLiveStreaming()).toThrowError('The library has not yet been initialized');
     });
 
-    it('should return error code 500', async () => {
-      await desktopPlatformMock.initializeWithContext(FrameContexts.sidePanel);
-
-      const promise = meeting.requestStopLiveStreaming();
-
-      const requestStopLiveStreamingMessage = desktopPlatformMock.findMessageByFunc('meeting.requestStopLiveStreaming');
-      expect(requestStopLiveStreamingMessage).not.toBeNull();
-
-      const callbackId = requestStopLiveStreamingMessage.id;
-      desktopPlatformMock.respondToMessage({
-        data: {
-          id: callbackId,
-          args: [{ errorCode: ErrorCode.INTERNAL_ERROR }, null],
-        },
-      } as DOMMessageEvent);
-
-      await expect(promise).rejects.toEqual({ errorCode: ErrorCode.INTERNAL_ERROR });
-    });
-
-    it('should successfully request start live streaming', async () => {
-      await desktopPlatformMock.initializeWithContext(FrameContexts.sidePanel);
-
-      const promise = meeting.requestStopLiveStreaming();
-
-      const requestStopLiveStreamingMessage = desktopPlatformMock.findMessageByFunc('meeting.requestStopLiveStreaming');
-      expect(requestStopLiveStreamingMessage).not.toBeNull();
-
-      const callbackId = requestStopLiveStreamingMessage.id;
-      desktopPlatformMock.respondToMessage({
-        data: {
-          id: callbackId,
-          args: [null, { isStreaming: false }],
-        },
-      } as DOMMessageEvent);
-
-      await expect(promise).resolves;
+    const allowedContexts = [FrameContexts.sidePanel];
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContext => allowedContext === context)) {
+        it('should successfully send the requestStartLiveStreaming message.', async () => {
+          await framelessPlatformMock.initializeWithContext(FrameContexts.sidePanel);
```

Comment:
We want code to pass the context here change `FrameContexts.sidePanel` => `context`